### PR TITLE
JtReader: Fix header uniformity and trailing whitespace

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7919">pipeline status</a> for feature branch [PR_7919](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7919). Pipeline [710001220](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/710001220) was triggered at [38be010](https://github.com/FreeCAD/FreeCAD/pull/7919/commits/38be0109028abf0299b0afe9f04ef40374ed6f4e). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/JtReader/App/JrJt/Context.h [hunk 1]
```diff
@@ -36,7 +36,7 @@ class Context
     uint8_t VersionMinor;
     uint8_t VersionMajor;
 
-    bool Endiness;
+    bool Endianness;
```
Comment (luzpaz) [inline src/Mod/JtReader/App/JrJt/Context.h:39]: Please review

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7919">pipeline status</a> for feature branch [PR_7919](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7919). Pipeline [710022476](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/710022476) was triggered at [bf93070](https://github.com/FreeCAD/FreeCAD/pull/7919/commits/bf930704bb48c03fb62f72d651ecda2a25b88aaf). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/JtReader/App/JrJt/Context.h (hunk 1 above)
Comment (donovaly) [reply src/Mod/JtReader/App/JrJt/Context.h:39]: Changing the name must also lead to further changes in the code where the bool is used. If the bool is not used, it should be removed. I only see that you changed the name, nothing else. Is that right?

Comment (luzpaz) [reply src/Mod/JtReader/App/JrJt/Context.h:39]: Confirmed, searching the whole codebase (again) shows this is the only place this bool is used.

Comment (luzpaz) [reply src/Mod/JtReader/App/JrJt/Context.h]: Removed in [f34b3c0](https://github.com/FreeCAD/FreeCAD/pull/7919/commits/f34b3c0ecf682743d613da460d48b0fe1b6566b7)

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7919">pipeline status</a> for feature branch [PR_7919](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7919). Pipeline [710659216](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/710659216) was triggered at [f34b3c0](https://github.com/FreeCAD/FreeCAD/pull/7919/commits/f34b3c0ecf682743d613da460d48b0fe1b6566b7). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**